src/utils.js

```diff
@@ -498,7 +498,7 @@ export const getDefaultFrontMatter = (params, existingTitles) => {
     }`
   }
   examplePermalink += `permalink`
-  if (collectionName)
+  if (!resourceRoomName)
```

Comment:
wdyt about writing a unit test for this!

Reply:
We currently don't have unit tests for the frontend - i think we might want to consider this when we start adding unit tests. For now, I've added a test for the request body being sent by the create request in https://github.com/isomerpages/isomercms-frontend/pull/776/commits/331e3686d08517c3d95640c2fecda5d62e363e4e, but we also don't do this anywhere else currently

Reply:
^actually we do have unit tests! it's under `src/__tests__` - i've updated it in [another](https://github.com/isomerpages/isomercms-frontend/pull/742) PR so it passes.

i guess like the main point of this is just to 1. check correctness 2. prevent regressions in the future!

Reply:
That makes sense, added unit tests for this util function in https://github.com/isomerpages/isomercms-frontend/pull/776/commits/2c118cd3cc904a844424527722485181b4a368db